Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [workspace]
 members = [
 	"base",
-	"bin_prot_checker",
 	"consensus",
 	"crypto",
-	"bin-prot",
-	"wire-type",
-	"wire-type-derive",
-	"test-fixtures",
-	"test-serialization",
+	# All crates related to bin prot wire protocol
+	"protocol/bin-prot",
```

Comment:
I think wildcard should be sufficient here

Reply:
we might also put non-crates such as docs

Reply:
How about using [SwaggerHub](https://support.smartbear.com/swaggerhub/docs/enterprise/release-notes.html)?